### DISCUSSION TITLE
rock/jazz instruments

### DISCUSSION THREAD

**Discussion by gerrybass** [musescore/MuseScore#14236] [Q&A]
i am missing the possibility to choose instruments like 'funk bass', 'guitar w distortion' as in MU3
or am i not finding the right path ...?

**Comment by MarcSabatella** [comment]
The *instruments* are simply electric guitar and electric bass.  To apply a specific *effect* to that instrument, see the Text palette.  I see distrot, but not "funk".  Probably that should be added too.  Eventually a means will be added to select sounds from within soudnfonts, which is what I assume you were really doing in MU3 (there was no "guitar w distortion" *instrument* there either).
